cli/command/service/formatter_test.go

```diff
@@ -65,25 +73,32 @@ bar
 		},
 		{
 			formatter.Context{Format: NewListFormat("raw", true)},
-			`id: id_baz
-id: id_bar
+			`id: 02_bar
+id: 01_baz
+id: 04_qux2
+id: 03_qux10
 `,
 		},
 		// Custom Format
 		{
 			formatter.Context{Format: NewListFormat("{{.Name}}", false)},
-			`baz
-bar
+			`bar
+baz
+qux2
+qux10
 `,
 		},
 	}
 
 	for _, testcase := range cases {
 		services := []swarm.Service{
 			{
-				ID: "id_baz",
+				ID: "01_baz",
```

Comment:
prefixing the ID's with numbers, so that we verify the list is sorted by the service _name_ not the service _ID_